cogs/help.py

```diff
@@ -124,15 +143,22 @@ def disable_all_buttons():
         else:
             disable_all_buttons()
             await interaction.edit_original_message(content = "Created!", view = confirm_view)
-            await self.create_help_thread(interaction)
+            created_thread = await self.create_help_thread(interaction)
+            await self.__launch_wait_for_message(created_thread, interaction)
 
 
 class HelpView(ui.View):
-    def __init__(self):
+    def __init__(self, bot: commands.Bot):
         super().__init__(timeout = None)
-        self.add_item(HelpButton("Nextcord", style = ButtonStyle.red, custom_id = "nextcord"))
-        self.add_item(HelpButton("Python", style = ButtonStyle.green, custom_id = "python"))
+        self.add_item(HelpButton(bot, "Nextcord", style = ButtonStyle.blurple, custom_id = "nextcord"))
+        self.add_item(HelpButton(bot, "Python", style = ButtonStyle.green, custom_id = "python"))
+
+    async def interaction_check(self, interaction: Interaction) -> bool:
+        if interaction.user.timeout is not None:
+            await interaction.send("You are currently timed out. Please wait until you are removed from it.", ephemeral=True)
```

Comment:
This should be worded better.

For example: `You are currently timed out. Please wait until your time out expires before trying again.`